casa/inventory/static/inventory/css/ui-revamp.css

```
@@ -1179,20 +1217,236 @@ body {
+  color: #8fa3ad;
+}
+
+html[data-theme="dark"] .table,
+html[data-theme="dark"] .list-group-item,
+html[data-theme="dark"] .nav-tabs,
+html[data-theme="dark"] .border,
+html[data-theme="dark"] .border-top,
+html[data-theme="dark"] .border-bottom {
+  --bs-table-color: var(--text);
+  --bs-table-border-color: var(--line);
+  border-color: var(--line) !important;
```

Stylelint (17.14.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)